Lib/imaplib.py

```diff
@@ -952,6 +949,12 @@ def _command(self, name, *args):
         for arg in args:
             if arg is None: continue
             if isinstance(arg, str):
+                if _Quoted_Invalid.search(arg):
+                    raise self.error('illegal cr or lf in argument')
+                if len(arg) > 2 and [arg[0], arg[-1]] == ['(', ')']:
+                    arg = arg[1:-1]
```

Comment:
This looks like it is removing the brackets, which seems undesirable. Compare with the original __checkquote_ implementation removed in commit f241afa.